PublicMapping/districtbuilder
# Add theme-ui and componentize login screen

## Overview

Adds [`theme-ui`](https://theme-ui.com/) and uses it to componentize the login screen

### Demo

![localhost_3003_login (4)](https://user-images.githubusercontent.com/4432106/81447607-f7401180-914a-11ea-81f4-987931bcacb8.png)
![localhost_3003_login (5)](https://user-images.githubusercontent.com/4432106/81447605-f6a77b00-914a-11ea-93da-d85caa20028d.png)


## Testing Instructions

- `scripts/update`
- `scripts/server`
- Go to the login screen
  - attempt to login with a missing user or incorrect password to see errors
  - verify login still works correctly

Closes #93
